packages/platform_linux/pubspec.yaml

```diff
@@ -14,6 +14,6 @@ dependencies:
   platform: ^3.1.0
 
 dev_dependencies:
-  ubuntu_lints: ^0.1.0
   mockito: ^5.4.2
   test: ^1.21.0
+  ubuntu_lints: ^0.1.0
```

Comment:
Just double-checking: `melos bootstrap` adds its own overrides to use the local package, right? So this version needs to be bumped manually once 0.2.0 gets released?